package.json

```diff
@@ -3,7 +3,7 @@
     "version": "0.7.0-preview",
     "displayName": "Project Auditor",
     "unity": "2018.4",
-    "description": "Project Auditor is an experimental static analysis tool for Unity Projects. This tool will analyze scripts and project settings of any Unity project and report a list a possible problems that might affect performance, memory and other areas. Note that this package is not officially supported by Unity and it is not on Unity's roadmap at this time.",
+    "description": "Project Auditor is an experimental static analysis tool that analyzes assets, settings and scripts of the Unity project and produces a report containing: Code and Settings Diagnostics, the last BuildReport and assets information. Note that this package is not officially supported by Unity and it is not on Unity's roadmap at this time.",
```

Comment:
I'm going to be that nit-picker and highlight that the Unity style guide uses the Oxford comma which would edit this to read as:

"Project Auditor is an experimental static analysis tool that analyzes assets, settings, and scripts of the Unity project and produces a report containing: Code and Settings Diagnostics, the last BuildReport, and assets information. Note that this package is not officially supported by Unity and it is not on Unity's roadmap at this time."

Reply:
Thanks for pointing that out. I'll fix it and merge the PR.